data/mods/Sorcerer/README.md

```diff
@@ -0,0 +1,101 @@
+Each time you level up, you can swap one spell you know with a spell of the same or lower level, so you are not locked in to your choices forever.<br>
+Inspired by Pathfinder and Dungeons and Dragons.
+
+## Contributing
+
+Please note that the code inside the folder generated_code is auto generated and should not be edited manually. Instead, use the python script in tools. You might have to adjust the paths used before you run them.<br>
+generated_code/spell_data_dump.txt contains data for all spells found and generated. You can copy that data into tools/spell_data.txt to edit properties of spells, disable spells, register sorcerer-specific spells, or manually alter the spell level of the spell.<br>
+I have not yet figured out a way to auto format the generated code, so you will have to deal with that yourself.<br>
+Note that while tools/spell_data.txt contains json data, it is a .txt file. This is to make sure CDDA doesn't confuse it for something it should be reading. The json is read by tools/generate_spell_levels.py.<br>
+Bloodline-specific stuff should if practical go into the bloodlines folders, either in an .json named after the bloodline, or in a folder with the same name. The exception is code for when you first pick the bloodline, which is currently located in pick_bloodline.json.
+
+## Balancing
+
+It seems that sorcerers will end up being more powerful than traditional Magiclysm wizards. I'm not yet sure if that is good or not. The mod might need a fair bit of nerfing.<br>
+One path forwards would be to give traditional Magiclysm wizards more toys depending on their school, like the school weapons, the caster level boosting items, or the druids wildshape.<br>
+Bloodlines should feel roughly equally powerful to each other, preferably.<br>
+My thinking is that if you are playing as a traditional Magiclysm wizard, you would level your main combat spell to around level 5 before you try to use it in combat, so a sorcerer should start with their main combat spell's effective spell level being around there as well. A sorcerer will likely gain levels in less in-game time than a wizard would, but would have to do more dangerous activities to do so.<br>
+The leveling up speed is supposed to be equal to the default settings for BombasticPerks.
+
+## Other things the mod contains
+
+Counterfeit Rune: This spell creates a small pebble infused with a sorcerer's power. It can trick spells that require runes to cast to think that you have the rune required. Does not work for crafting, but lets you cast animist spells.
+
+# Future plans and loose ideas
+
+Some more rebalancing and nerfing is probably needed. I'm open to suggestions. Perhaps your sorcerer level should only give half its level to you caster level? Perhaps higher level spells should have a reduced caster level bonus so that you start using them with a low level?
```

Comment:
```suggestion
Some more rebalancing and nerfing is probably needed. I'm open to suggestions. Perhaps your sorcerer level should only give half its level to your caster level? Perhaps higher level spells should have a reduced caster level bonus so that you start using them with a low level?
```